src/demo-app/tabs/tabs-demo.html

```diff
@@ -2,7 +2,7 @@
 
 <div class="demo-nav-bar">
   <nav md-tab-nav-bar aria-label="weather navigation links">
-    <a md-tab-link
+    <a md-tab-link md-ripple
```

Comment:
Sure, sounds good. If you haven't already, you might want to check out md-button's ripple. It's also an attr directive on an anchor.